kubernetes/pods/mongodb-helm/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    release: mongodb
+  name: mongodb
+  namespace: mongodb
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mongodb
+      release: mongodb
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: '/metrics'
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9001"
+      labels:
+        app: mongodb
+        release: mongodb
+    spec:
+      containers:
+      - env:
+        - name: MONGODB_ROOT_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: mongodb-root-password
+              name: mongodb
+        - name: MONGODB_USERNAME
+        - name: MONGODB_DATABASE
+        - name: MONGODB_EXTRA_FLAGS
+        image: docker.io/bitnami/mongodb:3.6.5
```

Comment:
Can we use the mongo image that the helm chart is using instead?  https://github.com/sekka1/cloud-public/tree/master/kubernetes/pods/mongodb-helm

I wont want to have to keep track of this bitnami image

Reply:
https://github.com/kubernetes/charts/blob/master/stable/mongodb/values.yaml#L1-L11

```
image:
  ## Bitnami MongoDB registry
  ##
  registry: docker.io
  ## Bitnami MongoDB image name
  ##
  repository: bitnami/mongodb
  ## Bitnami MongoDB image tag
  ## ref: https://hub.docker.com/r/bitnami/mongodb/tags/
  ##
  tag: 3.6.5
```